src/lib/design-system/themes.ts

```diff
@@ -0,0 +1,108 @@
+import {create} from './lib'
+
+export type Theme = typeof light
+
+export const light = create({
+  tokens: {
+    space: {
+      s: 10,
+      m: 16,
+      l: 24,
+    },
+    color: {
```

Comment:
Looking good. We probably want to add the colors from the existing `theme.ts` file